### DISCUSSION TITLE
Does margin-top or margin-bottom have an effect on inline elements?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#212] [Q&A] [answered]
In short.

**Comment by hongducdev** [comment, marked as answer]
Top and bottom margins do not affect inline elements because inline elements flow with content on the page. You can set left and right margins/padding on an inline element but not top or bottom because it would disrupt the flow of content. You can set margins on block (or inline-block but it will

**Comment by d4tdev** [comment]
No, it doesn’t affect the inline elements. Inline elements flow with the contents of the page.

**Comment by Preetiraj3697** [comment]
Top and bottom margins do not affect inline elements because inline elements flow with content on the page. You can set left and right margins/padding on an inline element but not top or bottom because it would disrupt the flow of content. You can set margins on block (or inline-block but it will only look right if you set the vertical align right) because block level elements disrupt the flow of content.
